cpp/oneapi/dal/backend/primitives/sparse_blas/gemv_dpc.cpp

```diff
@@ -0,0 +1,57 @@
+/*******************************************************************************
+* Copyright contributors to the oneDAL project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/backend/primitives/blas/misc.hpp"
+#include "oneapi/dal/backend/primitives/sparse_blas/gemv.hpp"
+
+namespace oneapi::dal::backend::primitives {
+
+template <typename Float>
+sycl::event gemv(sycl::queue& queue,
```

Comment:
I think we should better name the function sparse_gemv as long as it is defined in the same namespace with dense gemv.